pkg/credentialquery/errors.go

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright Avast Software. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package credentialquery
+
+// nolint: golint,nolintlint
+const (
+	module                               = "CRQ"
+	CredentialReaderNotSetError          = "CREDENTIAL_READER_NOT_SET"     //nolint: gosec
+	CredentialReaderReadFailedError      = "CREDENTIAL_READER_READ_FAILED" //nolint:gosec
```

Comment:
Reason for the `nolint: golint,nolintlint`? I'm guessing you did that because the linter requires a comment for each of those exported constants, and their purposes are pretty obvious already, but it's a good idea to at least add the comment explaining the reason for the nolint. Same thing for the `gosec` - in this case, it's a false positive as the linter flags anything with the word `credential` in it, but I still think it's a good idea to at least add `false positive` in the nolint comment to explain it, since it can look concerning to see a `gosec` warning being bypassed without an explanation. Also, one more minor thing - the spacing between the `nolint` and `gosec` should be consistent.

Reply:
(for consistency with the rest of the codebase, I would suggest just removing the `nolint: golint,nolintlint` and adding the export comments, even if it seems somewhat redundant)

Reply:
I think we should create a separate task for this. Comments should be meaningful, contains reasons why that error can happen, and if it is caused by wrong use of API, the comment should suggest a solution, e.g. "Please create DID before calling this function" etc. But this is very time-consuming, and I think should be done module by module.

Reply:
That's fair